ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace zadanie10
+{
+    internal class Program
+    {
+        static string Print(int[] n)
+        {
+            string str = "";
+            for (int i = 0; i < n.Length; i++)
+            {
+                str += $"{n[i]} ";
+            }
+            return Regex.Replace(str, ".{" + 20 + "}(?!$)", "$0\n");
```

Comment:
Зачем вам регулярные выражения? Полученная строка и так в нужном виде.
```suggestion
            return str;
```